mungegithub/mungers/submit-queue.go

```diff
@@ -783,7 +783,7 @@ func (sq *SubmitQueue) getMetaData() []byte {
 const (
 	unknown                 = "unknown failure"
 	noCLA                   = "PR is missing CLA label; needs one of " + claYesLabel + ", " + cncfClaYesLabel + " or " + claHumanLabel
-	noLGTM                  = "PR does not have LGTM."
+	noLGTM                  = "PR does not have LGTM or Approved Label (needs at least one)."
```

Comment:
any chance while you are here you'd be willing to use `lgtmLabel` and `approvedLabel` in the string?